dockers/llm.rag.service/common.py

```diff
@@ -290,6 +303,12 @@ def prompt_template_for_text_to_sql():
         "which table. Only use the following tables: {table_info}."
         "If there is a ticket ID in the question, ensure that you maintain "
         "the exact ticket ID in the query."
+        "If the query retrieves specific ticket details, **always include the ticket_id column** in the result set, "
+        "even if the user did not explicitly ask for it. This ensures the ticket ID is present in ticket-related queries."
+        "However, if the query is an aggregate function (such as counting all tickets), omit the ticket ID. "
+        "Always include `ticket_id` in ticket-related queries. **Do not use `ticket_url` unless explicitly requested.**"
+        "Do not make any references to the SQL query or the SQL result in your answer."
```

Comment:
This allows to add ticket_id to the query. For us, ticket_url would be better, but changing ticket_id to ticket_url doesn't work as expected.

Short term we can build ticket url in code, but long term need to figure it out.